### ISSUE TITLE
Flashing magnetic line

### ISSUE THREAD

**Issue by sfederici** [jmoenig/Snap--Build-Your-Own-Blocks#916]
When shift-clicking a non-hat-block (or stack) in the script area a flashing magnetic line shows up above the (first) block. The line doesn't move when the block/stack is dragged.

**Comment by brianharvey** [comment]
Sounds like you have keyboard editing turned on.

**Comment by cycomachead** [comment]
> The line doesn't move when the block/stack is dragged.

This does seem like a bit of a bug / UI annoyance.

**Comment by sfederici** [comment]
> Sounds like you have keyboard editing turned on.

Yes I do. When keyboard editing is off the magnetic line doesn't show up when I shift-click.

**Comment by sfederici** [comment]
The same (the magnetic lines do not follow the blocks) happens when cleaning up the script area.
